skins/agdeportes.scss

```
@@ -0,0 +1,9 @@
+.agdeportes {
+  --primary-color: #fde25;
```

Please correct your hexademical colour.
```suggestion
  --primary-color: #fde250;
```